org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r6 = new java.util.HashSet >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : specialinvoke $r6.<java.util.HashSet: void <init>()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : specialinvoke $r6.<java.util.HashSet: void <init>()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r7 = $r6
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r7 = $r6 >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r8 = null
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r8 = null >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r8 = null >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r8 = null >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r8 = virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: java.sql.ResultSet getResultSet(java.util.Set,java.lang.String,java.lang.String,java.lang.String)>(r1, r3, r4, r5)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r8 = virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: java.sql.ResultSet getResultSet(java.util.Set,java.lang.String,java.lang.String,java.lang.String)>(r1, r3, r4, r5) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r8 = virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: java.sql.ResultSet getResultSet(java.util.Set,java.lang.String,java.lang.String,java.lang.String)>(r1, r3, r4, r5) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r8 = virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: java.sql.ResultSet getResultSet(java.util.Set,java.lang.String,java.lang.String,java.lang.String)>(r1, r3, r4, r5) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $z0 = interfaceinvoke r8.<java.sql.ResultSet: boolean next()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $z0 = interfaceinvoke r8.<java.sql.ResultSet: boolean next()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $z0 = interfaceinvoke r8.<java.sql.ResultSet: boolean next()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $z0 = interfaceinvoke r8.<java.sql.ResultSet: boolean next()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if $z0 == 0
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if $z0 == 0 >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r14 = new org.apache.ambari.server.controller.internal.ResourceImpl
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if $z0 == 0 >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if r8 == null
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if $z0 == 0 >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if $z0 == 0 >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r12 = $r51 >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if r8 == null
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r12 = $r51 >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if r8 == null >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke r8.<java.sql.ResultSet: void close()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if r8 == null >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: void close()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if r8 == null >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r52 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r13 = $r52 >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r53 = staticinvoke <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: org.apache.commons.logging.Log access$000()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r53 = staticinvoke <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: org.apache.commons.logging.Log access$000()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke $r53.<org.apache.commons.logging.Log: void error(java.lang.Object,java.lang.Throwable)>("Exception while closing ResultSet", r13)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke $r53.<org.apache.commons.logging.Log: void error(java.lang.Object,java.lang.Throwable)>("Exception while closing ResultSet", r13) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: void close()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: void close()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : throw r12
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke r8.<java.sql.ResultSet: void close()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : goto [?= virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: void close()>()]
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke r8.<java.sql.ResultSet: void close()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r52 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : goto [?= virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: void close()>()] >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: void close()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r46 = $r45 >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r47 = staticinvoke <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: org.apache.commons.logging.Log access$000()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r46 = $r45 >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r47 = staticinvoke <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: org.apache.commons.logging.Log access$000()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $z4 = interfaceinvoke $r47.<org.apache.commons.logging.Log: boolean isDebugEnabled()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r47 = staticinvoke <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: org.apache.commons.logging.Log access$000()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $z4 = interfaceinvoke $r47.<org.apache.commons.logging.Log: boolean isDebugEnabled()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if $z4 == 0
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $z4 = interfaceinvoke $r47.<org.apache.commons.logging.Log: boolean isDebugEnabled()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if $z4 == 0 >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r48 = staticinvoke <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: org.apache.commons.logging.Log access$000()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if $z4 == 0 >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r10 = staticinvoke <java.util.Collections: java.util.Set emptySet()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if $z4 == 0 >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r10 = staticinvoke <java.util.Collections: java.util.Set emptySet()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if r8 == null
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r10 = staticinvoke <java.util.Collections: java.util.Set emptySet()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if r8 == null >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke r8.<java.sql.ResultSet: void close()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if r8 == null >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: void close()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if r8 == null >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r49 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r11 = $r49 >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r50 = staticinvoke <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: org.apache.commons.logging.Log access$000()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r50 = staticinvoke <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: org.apache.commons.logging.Log access$000()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke $r50.<org.apache.commons.logging.Log: void error(java.lang.Object,java.lang.Throwable)>("Exception while closing ResultSet", r11)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke $r50.<org.apache.commons.logging.Log: void error(java.lang.Object,java.lang.Throwable)>("Exception while closing ResultSet", r11) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: void close()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: void close()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : return r10
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke r8.<java.sql.ResultSet: void close()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : goto [?= virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: void close()>()]
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke r8.<java.sql.ResultSet: void close()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r49 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : goto [?= virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: void close()>()] >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: void close()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r48 = staticinvoke <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: org.apache.commons.logging.Log access$000()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke $r48.<org.apache.commons.logging.Log: void debug(java.lang.Object,java.lang.Throwable)>("Caught exception getting resource.", r46)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r48 = staticinvoke <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: org.apache.commons.logging.Log access$000()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke $r48.<org.apache.commons.logging.Log: void debug(java.lang.Object,java.lang.Throwable)>("Caught exception getting resource.", r46) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r10 = staticinvoke <java.util.Collections: java.util.Set emptySet()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke $r48.<org.apache.commons.logging.Log: void debug(java.lang.Object,java.lang.Throwable)>("Caught exception getting resource.", r46) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if r8 == null >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke r8.<java.sql.ResultSet: void close()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if r8 == null >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: void close()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : if r8 == null >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r42 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r43 = $r42 >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r44 = staticinvoke <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: org.apache.commons.logging.Log access$000()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r44 = staticinvoke <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: org.apache.commons.logging.Log access$000()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke $r44.<org.apache.commons.logging.Log: void error(java.lang.Object,java.lang.Throwable)>("Exception while closing ResultSet", r43)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke $r44.<org.apache.commons.logging.Log: void error(java.lang.Object,java.lang.Throwable)>("Exception while closing ResultSet", r43) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: void close()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: void close()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : goto [?= return r7]
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : goto [?= return r7] >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : return r7
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke r8.<java.sql.ResultSet: void close()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : goto [?= virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: void close()>()]
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke r8.<java.sql.ResultSet: void close()>() >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r42 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : goto [?= virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: void close()>()] >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: void close()>()
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r14 = new org.apache.ambari.server.controller.internal.ResourceImpl >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r14 = new org.apache.ambari.server.controller.internal.ResourceImpl >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r14 = new org.apache.ambari.server.controller.internal.ResourceImpl >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r15 = <org.apache.ambari.server.controller.spi.Resource$Type: org.apache.ambari.server.controller.spi.Resource$Type TaskAttempt>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r15 = <org.apache.ambari.server.controller.spi.Resource$Type: org.apache.ambari.server.controller.spi.Resource$Type TaskAttempt> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r15 = <org.apache.ambari.server.controller.spi.Resource$Type: org.apache.ambari.server.controller.spi.Resource$Type TaskAttempt> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r15 = <org.apache.ambari.server.controller.spi.Resource$Type: org.apache.ambari.server.controller.spi.Resource$Type TaskAttempt> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : specialinvoke $r14.<org.apache.ambari.server.controller.internal.ResourceImpl: void <init>(org.apache.ambari.server.controller.spi.Resource$Type)>($r15)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : specialinvoke $r14.<org.apache.ambari.server.controller.internal.ResourceImpl: void <init>(org.apache.ambari.server.controller.spi.Resource$Type)>($r15) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : specialinvoke $r14.<org.apache.ambari.server.controller.internal.ResourceImpl: void <init>(org.apache.ambari.server.controller.spi.Resource$Type)>($r15) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : specialinvoke $r14.<org.apache.ambari.server.controller.internal.ResourceImpl: void <init>(org.apache.ambari.server.controller.spi.Resource$Type)>($r15) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r9 = $r14
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r9 = $r14 >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r9 = $r14 >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : r9 = $r14 >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r16 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_CLUSTER_NAME_PROPERTY_ID>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r16 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_CLUSTER_NAME_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r16 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_CLUSTER_NAME_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r16 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_CLUSTER_NAME_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : staticinvoke <org.apache.ambari.server.controller.internal.BaseProvider: boolean setResourceProperty(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.lang.Object,java.util.Set)>(r9, $r16, r2, r1)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : staticinvoke <org.apache.ambari.server.controller.internal.BaseProvider: boolean setResourceProperty(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.lang.Object,java.util.Set)>(r9, $r16, r2, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : staticinvoke <org.apache.ambari.server.controller.internal.BaseProvider: boolean setResourceProperty(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.lang.Object,java.util.Set)>(r9, $r16, r2, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : staticinvoke <org.apache.ambari.server.controller.internal.BaseProvider: boolean setResourceProperty(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.lang.Object,java.util.Set)>(r9, $r16, r2, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r17 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_WORKFLOW_ID_PROPERTY_ID>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r17 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_WORKFLOW_ID_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r17 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_WORKFLOW_ID_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r17 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_WORKFLOW_ID_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : staticinvoke <org.apache.ambari.server.controller.internal.BaseProvider: boolean setResourceProperty(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.lang.Object,java.util.Set)>(r9, $r17, r3, r1)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : staticinvoke <org.apache.ambari.server.controller.internal.BaseProvider: boolean setResourceProperty(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.lang.Object,java.util.Set)>(r9, $r17, r3, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : staticinvoke <org.apache.ambari.server.controller.internal.BaseProvider: boolean setResourceProperty(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.lang.Object,java.util.Set)>(r9, $r17, r3, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : staticinvoke <org.apache.ambari.server.controller.internal.BaseProvider: boolean setResourceProperty(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.lang.Object,java.util.Set)>(r9, $r17, r3, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r18 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r18 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r18 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r18 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r19 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_JOB_ID_PROPERTY_ID>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r19 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_JOB_ID_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r19 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_JOB_ID_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r19 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_JOB_ID_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r18.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setString(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r19, r8, r1)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r18.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setString(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r19, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r18.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setString(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r19, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r18.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setString(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r19, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r20 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r20 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r20 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r20 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r21 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_ID_PROPERTY_ID>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r21 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_ID_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r21 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_ID_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r21 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_ID_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r20.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setString(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r21, r8, r1)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r20.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setString(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r21, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r20.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setString(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r21, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r20.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setString(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r21, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r22 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r22 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r22 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r22 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r23 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_TYPE_PROPERTY_ID>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r23 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_TYPE_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r23 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_TYPE_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r23 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_TYPE_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r22.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setString(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r23, r8, r1)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r22.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setString(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r23, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r22.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setString(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r23, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r22.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setString(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r23, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r24 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r24 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r24 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r24 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r25 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_START_TIME_PROPERTY_ID>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r25 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_START_TIME_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r25 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_START_TIME_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r25 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_START_TIME_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r24.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r25, r8, r1)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r24.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r25, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r24.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r25, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r24.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r25, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r26 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r26 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r26 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r26 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r27 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_FINISH_TIME_PROPERTY_ID>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r27 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_FINISH_TIME_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r27 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_FINISH_TIME_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r27 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_FINISH_TIME_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r26.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r27, r8, r1)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r26.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r27, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r26.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r27, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r26.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r27, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r28 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r28 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r28 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r28 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r29 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_MAP_FINISH_TIME_PROPERTY_ID>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r29 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_MAP_FINISH_TIME_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r29 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_MAP_FINISH_TIME_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r29 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_MAP_FINISH_TIME_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r28.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r29, r8, r1)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r28.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r29, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r28.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r29, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r28.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r29, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r30 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r30 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r30 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r30 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r31 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_SHUFFLE_FINISH_TIME_PROPERTY_ID>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r31 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_SHUFFLE_FINISH_TIME_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r31 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_SHUFFLE_FINISH_TIME_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r31 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_SHUFFLE_FINISH_TIME_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r30.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r31, r8, r1)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r30.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r31, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r30.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r31, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r30.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r31, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r32 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r32 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r32 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r32 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r33 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_SORT_FINISH_TIME_PROPERTY_ID>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r33 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_SORT_FINISH_TIME_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r33 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_SORT_FINISH_TIME_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r33 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_SORT_FINISH_TIME_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r32.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r33, r8, r1)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r32.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r33, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r32.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r33, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r32.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r33, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r34 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r34 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r34 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r34 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r35 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_INPUT_BYTES_PROPERTY_ID>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r35 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_INPUT_BYTES_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r35 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_INPUT_BYTES_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r35 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_INPUT_BYTES_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r34.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r35, r8, r1)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r34.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r35, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r34.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r35, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r34.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r35, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r36 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r36 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r36 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r36 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r37 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_OUTPUT_BYTES_PROPERTY_ID>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r37 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_OUTPUT_BYTES_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r37 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_OUTPUT_BYTES_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r37 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_OUTPUT_BYTES_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r36.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r37, r8, r1)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r36.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r37, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r36.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r37, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r36.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setLong(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r37, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r38 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r38 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r38 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r38 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r39 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_STATUS_PROPERTY_ID>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r39 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_STATUS_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r39 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_STATUS_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r39 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_STATUS_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r38.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setString(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r39, r8, r1)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r38.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setString(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r39, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r38.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setString(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r39, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r38.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setString(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r39, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r40 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r40 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r40 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r40 = r0.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher: org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider this$0> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r41 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_LOCALITY_PROPERTY_ID>
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r41 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_LOCALITY_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r41 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_LOCALITY_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r41 = <org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: java.lang.String TASK_ATTEMPT_LOCALITY_PROPERTY_ID> >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r40.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setString(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r41, r8, r1)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r40.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setString(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r41, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r40.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setString(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r41, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : virtualinvoke $r40.<org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider: void setString(org.apache.ambari.server.controller.spi.Resource,java.lang.String,java.sql.ResultSet,java.util.Set)>(r9, $r41, r8, r1) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke r7.<java.util.Set: boolean add(java.lang.Object)>(r9)
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke r7.<java.util.Set: boolean add(java.lang.Object)>(r9) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke r7.<java.util.Set: boolean add(java.lang.Object)>(r9) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : interfaceinvoke r7.<java.util.Set: boolean add(java.lang.Object)>(r9) >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : goto [?= $z0 = interfaceinvoke r8.<java.sql.ResultSet: boolean next()>()]
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : goto [?= $z0 = interfaceinvoke r8.<java.sql.ResultSet: boolean next()>()] >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r45 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : goto [?= $z0 = interfaceinvoke r8.<java.sql.ResultSet: boolean next()>()] >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $r51 := @caughtexception
org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : goto [?= $z0 = interfaceinvoke r8.<java.sql.ResultSet: boolean next()>()] >>>>>>>> org.apache.ambari.server.controller.internal.TaskAttemptResourceProvider$PostgresTaskAttemptFetcher.fetchTaskAttemptDetails : $z0 = interfaceinvoke r8.<java.sql.ResultSet: boolean next()>()
